src/complexity_measures/statistical_complexity.jl

```diff
@@ -0,0 +1,192 @@
+using Distances
+
+export StatisticalComplexity, maximum_complexity_entropy, minimum_complexity_entropy
+
+evaluate(distance::SemiMetric, a::Probabilities, b::Probabilities) = Distances.evaluate(distance, vec(a), vec(b))
```

Comment:
Without having tested (I'm not at the computer at the moment), I would guess that this line is slowing the algorithm down significantly. Converting to vectors here is unnecessary - the vectors already exist as fields of the `Probabilities` instances. Simply do `Distances.evaluate(a.p, b.p)`.

Reply:
Good point, fixed that! But it doesn't seem to be the (only) cause. Even with this, the entropy calculation is 30x faster than in python for `m=10`, while the complexity calculation is only 3x faster in julia.